src/sprites/rendered-target.js

```diff
@@ -840,6 +840,7 @@ class RenderedTarget extends Target {
      * @return {boolean} True iff touching a clone of the sprite.
      */
     isTouchingSprite (spriteName) {
+        spriteName = StringUtil.stringify(spriteName);
```

Comment:
Was there a particular reason you chose `StringUtil.stringify` here? I think this one should also be `Cast.toString`

Reply:
I was thinking I could use a util that already was required in the VM since it's such a small change, but in retrospect I was probably overthinking it. I'll add `Cast` and use `toString`.